lib/shopify_api/resources/fulfillment.rb

```diff
@@ -9,5 +9,26 @@ def order_id
     def cancel; load_attributes_from_response(post(:cancel, {}, only_id)); end
     def complete; load_attributes_from_response(post(:complete, {}, only_id)); end
     def open; load_attributes_from_response(post(:open, {}, only_id)); end
+
+    def self.create(attributes = {})
+      fulfillmentV2 = FulfillmentV2.create(attributes)
+      fulfillment = Fulfillment.new
+      fulfillment.load(fulfillmentV2.attributes, false, true)
+      fulfillment
+    end
+
+    def save
```

Comment:
We still need to support the ability to create fulfillments using the old endpoint, as well as the new fulfillment create:

<img width="401" alt="Screen Shot 2019-12-18 at 11 56 37 AM" src="https://user-images.githubusercontent.com/31415382/71289112-4b805b80-233a-11ea-8512-bee3f507b4fe.png">

We can do so by inspecting the passed in params and deciding if the client intended to use the legacy/new endpoint.